portfolio/src/main/webapp/script.js

```diff
@@ -31,3 +31,10 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+async function addMotivationalMessage(){
+  const responseFromServer = await fetch('/hello');
+  const textFromResponse = await responseFromServer.text();
```

Comment:
Do you need a second await here? Isn't the await on the responseFromServer sufficient?

Reply:
I believe both awaits are necessary because the first one tells JavaScript to wait for the response from the async function and the second one tells JavaScript to wait for the text content included in the response.